pzsd_bot/settings.py

```diff
@@ -87,6 +89,18 @@ class _PointsSettings(EnvSettings):
         re.IGNORECASE,
     )
 
+    @property
+    def page_buttons(self) -> list[PaginatorButton]:
```

Comment:
why isn't this just a constant

Reply:
I tried that, turns out if you don't generate new button objects for each leaderboard, the leaderboards will all share the same button objects, and hitting the button on one leaderboard will affect them all.